### ISSUE TITLE
0.9.1 and 0.9.2 weren't tagged

### ISSUE THREAD

**Issue by zenspider** [jimweirich/rake#56]
Maybe you should be using hoe with the git plugin?

**Comment by quix** [comment]
I verified these against the released gems:

```
rake-0.9.1: 86db7104b256956cb6807c3d38c6c8d1c50fcab8
rake-0.9.2: b2a8487575969d123d38ad8440ccfb860c060564
```

Incidentally I found a junk test file named "RRR" in the root directory of both released gems (#65). Due to a globbing oversight, we now know Jim's secret naming convention for test files :)

**Comment by zenspider** [comment]
Looks like they were added.
